src/core/sys/posix/setjmp.d

```diff
@@ -167,6 +167,12 @@ else version( FreeBSD )
         enum _JBLEN = 5;
         struct _jmp_buf { c_long[_JBLEN + 1] _jb; }
     }
+    else version( AArch64 )
+    {
+        enum _JBLEN = 31;
+        // __int128_t
+        struct _jmp_buf { long[2][_JBLEN + 1] _jb; };
+    }
```

Comment:
Everything after the first file should go upstream at dlang/druntime, submit this there instead.